network/network_common.go

```diff
@@ -9,10 +14,10 @@ import (
 
 // Network holds network metadata about the host
 type Network struct {
-	// IpAddress is the ipv4 address for the host
-	IpAddress string
-	// IpAddressv6 is the ipv6 address for the host
-	IpAddressv6 string
+	// IPAddress is the ipv4 address for the host
+	IPAddress string
+	// IPAddressv6 is the ipv6 address for the host
+	IPAddressv6 string
```

Comment:
Will this impact API use in downstream project(s)? If so, we will need to plan for the transition.

Reply:
Yes but since we're changing the whole API the `Network` struct (and the equivalents in other packages) will be removed anyway (to be replaced with the `network.Info` struct, cf the RFC)...

Reply:
That's fair but this will be applied to the main branch of the repo and break API if merged. Maybe we want to put the breaking changes in the diff after it's moved?

Reply:
Ok I see your point, what do you think is the best way forward then ? 
I guess the easiest would be to merge these changes in a branch instead of main (and use that branch instead of main when importing into the agent) ?

Reply:
I removed the breaking changes and used `//nolint` instead.